index.bs

```diff
@@ -285,6 +287,18 @@ Try to construct a table with the columns of the properties of <var>tabularData<
 
 Perform Logger("warn", <var>data</var>).
 
+<h4 id="dir" oldids="dir-data,dom-console-dir" method for="console">dir(<var>data</var>)</h4>
+
+Let <var>object</var> be an potentially-interactive representation of <var>data</var> treated as a JavaScript object. Perform Printer("log", <var>«object»</var>).
+
+<h4 id="dirxml" oldids="dirxml-data,dom-console-dirxml" method for="console">dirxml(...<var>data</var>)</h4>
+
+<emu-alg>
```

Comment:
This algorithm doesn't seem to process formatting specifiers, but Chrome does and Firefox does not, hmm. Maybe the best path is to leave as-is in this PR but open a separate issue to discuss?